akka-docs/src/main/paradox/general/configuration.md

```diff
@@ -57,7 +57,12 @@ one library in a `reference.conf` of another library.
 Akka requires a license key for use in production. Free keys can be obtained at [https://akka.io/key](https://akka.io/key).
 Add the key to the configuration property `akka.license-key`.
 
-For local development, Akka can be used without a key, but be aware of that the `ActorSystem` will terminate after a while when a key isn't configured.
+For local development, Akka can be used without a key, but be aware of that the `ActorSystem` will terminate after
+a while when a key isn't configured.
+
+If the license key has expired when the `ActorSystem` is started the system will terminate after a while.
+The expiry date is exposed by @apidoc[akka.actor.typed.ActorSystem] {scala="#licenseKeyExpiry" java="#getLicenseKeyExpiry"}
+so that you can write a test to remind yourself of that it is time for renewal before it has expired.
```

Comment:
Not important, but `of that` not needed.

```suggestion
For local development, Akka can be used without a key, but be aware that the `ActorSystem` will terminate after
a while when a key isn't configured.

If the license key has expired when the `ActorSystem` is started the system will terminate after a while.
The expiry date is exposed by @apidoc[akka.actor.typed.ActorSystem] {scala="#licenseKeyExpiry" java="#getLicenseKeyExpiry"}
so that you can write a test to remind yourself that it is time for renewal before it has expired.
```